src/AccentColor/AccentColorManager.vala

```diff
@@ -151,7 +151,12 @@ public class Gala.AccentColorManager : Object {
             return null;
         }
 
-        return metadata.get_tag_string (TAG_ACCENT_COLOR);
+        try {
+            return metadata.try_get_tag_string (TAG_ACCENT_COLOR);
+        } catch (Error e) {
+            warning (e.message);
+            return null;
```

Comment:
Instead of return null here should we return a fallback color?

Reply:
No, this function is used when getting accent color from wallpaper, it tries to get it from metadata and if it fails (returns null) update_accent_color selects accent color based on a picture itself.